### ISSUE TITLE
How to link volumes

### ISSUE THREAD

**Issue by davidcittadini** [linuxkit/linuxkit#2557]
I am using qeum with LinuxKit and there does not seem to be an obvious way to use LinuxKit, either by using the linuxkit command or the YML file, to do the same thing that "volumes" or "mount" does with Docker.  So, how does one use LinuxKit to dynamically or statically mount an external volume so that it is available inside the container?

**Comment by xtuc** [comment]
(i'm not an expert, just trying to help here)

- Do you want to mount a volume from your host inside your VM?
You should find the anwser in the qemu-kvm documentation. You can attach a volume as a storage device and `mount` it in your VM.

- Do you want to share a directory from inside your VM to some containers?
Use the `bind` instruction as describe in the [YAML spec](https://github.com/moby/tool/blob/master/docs/yaml.md#image-specification).

**Comment by davidcittadini** [comment]
That helped a lot, thanks!!  However, I still don't have the answer 100%.

I can see how using the `bind` instruction binds a directory inside the VM to inside the container, that works well.  However, with regards to mounting a volume from my host to inside the VM, I still cannot see:

- How to do that via qemu-kvm.  The qemu-kvm documentation has complex instructions on how to perform a Filesystem Passthrough to enable sharing of files on the host inside the VM.  I could not get any of the instructions to work with linuxkit qemu.

- The `Docker run` command has this simple `-v` option that enables you to bind host directories directly to the container.  There does not seem to be any option in perform the same function with the linuxkit command.

So, in summary, what I would like to do is mount a volume from the host to inside the VM but there does not seem to be an easy option to do this.
